tests/Oryx.Integration.Tests/DotNetCoreEndToEndTests.cs

```diff
@@ -278,7 +276,134 @@ public async Task CanBuildAndRun_NetCore21WebApp_UsingDefaultPublishOutputDirect
+                .ToString();
+            var runtimeImageScript = new ShellScriptBuilder()
+                .AddCommand(
+                $"oryx -appPath {appDir} -runFromPath /tmp/output -bindPort {ContainerPort}")
+                .AddCommand(DefaultStartupFilePath)
+                .ToString();
+
+            await EndToEndTestHelper.BuildRunAndAssertAppAsync(
+                NetCoreApp21WebApp,
+                _output,
+                volume,
+                "/bin/sh",
+                new[]
+                {
+                        "-c",
+                        buildImageScript
+                },
+                $"oryxdevms/dotnetcore-2.1",
+                ContainerPort,
+                "/bin/sh",
+                new[]
+                {
+                        "-c",
+                        runtimeImageScript
+                },
+                async (hostPort) =>
+                {
+                    var data = await _httpClient.GetStringAsync($"http://localhost:{hostPort}/");
+                    Assert.Contains("Hello World!", data);
+                });
+        }
+
+        [Fact]
+        public async Task CanBuildAndRun_MultiWebAppRepo_UsingDefaultPublishOutputDirectory()
+        {
+            // Arrange
+            var appName = "MultiWebAppRepo";
+            var hostDir = Path.Combine(_hostSamplesDir, "DotNetCore", appName);
+            var volume = DockerVolume.CreateMirror(hostDir);
+            var repoDir = volume.ContainerDir;
+            var setProjectEnvVariable = "export PROJECT=src/WebApp1/WebApp1.csproj";
+            var appOutputDir = $"{repoDir}/myoutputdir";
+            var buildImageScript = new ShellScriptBuilder()
+                .AddCommand(setProjectEnvVariable)
+                .AddCommand($"oryx build {repoDir}")
+                .ToString();
+            var runtimeImageScript = new ShellScriptBuilder()
+                .AddCommand(
+                $"oryx -appPath {repoDir} -bindPort {ContainerPort}")
+                .AddCommand(DefaultStartupFilePath)
+                .ToString();
+
+            await EndToEndTestHelper.BuildRunAndAssertAppAsync(
+                appName,
+                _output,
+                volume,
+                "/bin/sh",
+                new[]
+                {
+                    "-c",
+                    buildImageScript
+                },
+                $"oryxdevms/dotnetcore-2.1",
+                ContainerPort,
+                "/bin/sh",
+                new[]
+                {
+                    "-c",
+                    runtimeImageScript
+                },
+                async (hostPort) =>
+                {
+                    var data = await _httpClient.GetStringAsync($"http://localhost:{hostPort}/");
+                    Assert.Contains("Hello World! from WebApp1", data);
+                });
+        }
+
+        [Fact]
+        public async Task CanBuildAndRunWebApp_UsingDefaultPublishOutputDirectory_AndOutputIsZipped()
```

Comment:
For reference, example output from build and runtime images:

**Build**
```
Build orchestrated by Microsoft Oryx, https://github.com/Microsoft/Oryx
You can report issues at https://github.com/Microsoft/Oryx/issues

Oryx Version      : 0.2.unspecified, Commit: unspecified
Build Operation ID: |Ad8hbsfqNt4=.d8d09ca6_


Source directory     : /oryxtests/MultiWebAppRepo
Destination directory: /oryxtests/MultiWebAppRepo/oryx_publish_output


Using .NET Core SDK Version: 2.1.700

Restoring packages for 'src/WebApp1/WebApp1.csproj'...

  Restore completed in 220.24 ms for /oryxtests/MultiWebAppRepo/src/Greeter/Greeter.csproj.
  Restore completed in 543.1 ms for /oryxtests/MultiWebAppRepo/src/WebApp1/WebApp1.csproj.

Publishing 'src/WebApp1/WebApp1.csproj' to directory '/tmp/puboutput/publish'...

Microsoft (R) Build Engine version 16.1.76+g14b0a930a7 for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.

  Restore completed in 45.27 ms for /oryxtests/MultiWebAppRepo/src/Greeter/Greeter.csproj.
  Restore completed in 50.66 ms for /oryxtests/MultiWebAppRepo/src/WebApp1/WebApp1.csproj.
  Greeter -> /oryxtests/MultiWebAppRepo/src/Greeter/bin/Release/netcoreapp2.1/Greeter.dll
  WebApp1 -> /oryxtests/MultiWebAppRepo/src/WebApp1/bin/Release/netcoreapp2.1/MyWebApp.dll
  WebApp1 -> /tmp/puboutput/publish/

Removing existing manifest file
Creating a manifest file...
Manifest file created.

Done in  sec(s).
```

**Runtime**
```
Oryx Version : 0.2.unspecified, Commit: unspecified

Found publish output directory at '/oryxtests/MultiWebAppRepo/oryx_publish_output'. Using it's contents for running the app.
Build Operation ID: |Ad8hbsfqNt4=.d8d09ca6_
Building the startup script...
Read build manifest file and found output has been zipped, so adding script to extract it...
Writing output script to 'run.sh'

Running the script...
Found 'oryx_output.tar.gz' under '/oryxtests/MultiWebAppRepo/oryx_publish_output'. Extracting it's contents into it...
Done.
Deleting the file 'oryx_output.tar.gz'...
Done.
Finding the startup file name...
Found the startup file 'MyWebApp.dll'
Running the command 'dotnet 'MyWebApp.dll''...

Hosting environment: Production
Content root path: /oryxtests/MultiWebAppRepo/oryx_publish_output
Now listening on: http://[::]:3000
Application started. Press Ctrl+C to shut down.
info: Microsoft.AspNetCore.Hosting.Internal.WebHost[1]
      Request starting HTTP/1.1 GET http://localhost:33051/
info: Microsoft.AspNetCore.Hosting.Internal.WebHost[2]
      Request finished in 18.0374ms 200
```